packages/suite/src/actions/wallet/coinmarket/coinmarketCommonActions.ts

```diff
@@ -186,3 +195,41 @@ export const setLoading = (
 export const loadInvityData = (): CoinmarketCommonAction => ({
     type: COINMARKET_COMMON.LOAD_DATA,
 });
+
+export const convertDrafts = () => (dispatch: Dispatch, getState: GetState) => {
+    const { accounts, formDrafts, settings } = getState().wallet;
+    const formDraftKeys = Object.keys(formDrafts);
+
+    formDraftKeys.forEach(formDraftKey => {
+        const [prefix, accountKey] = parseFormDraftKey(formDraftKey);
+        const relatedAccount = accounts.find(({ key }) => key === accountKey);
+
+        if (!relatedAccount || !hasNetworkFeatures(relatedAccount, 'amount-unit')) {
+            return;
+        }
+
+        type FormState = { cryptoInput?: string; outputs?: Output[] };
```

Comment:
Perhaps the type should be defined outside of `convertDrafts()`

Reply:
The type was meant to be relevant only to this method. However, if you think it should be declared outside, I can change it.

Reply:
Yes please, just place it above the method. Declaring types inside of methods seems strange to me and we are not doing it anywhere :(

Reply:
Sure, fixed.